mobile.go

```diff
@@ -104,7 +105,10 @@ func doUpdateMobile(url string, out *os.File, updater Updater) error {
 	}
 
 	req.Header.Add("Accept-Encoding", "gzip")
-	pro.PrepareForFronting(req)
+
+	frontedURL := *req.URL
+	frontedURL.Host = "d2yl1zps97e5mx.cloudfront.net"
+	proxied.PrepareForFronting(req, frontedURL.String())
```

Comment:
@myleshorton Is the POST request `go-update` makes to see if an update is available actually idempotent? I just tried switching to using `proxied.ParallelPreferChained' and I get this:

```
                         E  Apr 15 17:58:39.203 - 0m10s ERROR go-update.check: check.go:170 Error submitting update request: Post https://update.getlantern.org/update: Use ParallelPreferChained
                             for non-idempotent method [error=Error submitting update request: %v error_location=github.com/getlantern/flashlight/vendor/github.com/getlantern/go-update/check.(*
                            Params).CheckForUpdate (check.go:170) error_op=Post error_text=Error submitting update request: Post https://update.getlantern.org/update: Use ParallelPreferChained
                            for non-idempotent method error_type=errors.Error http_method=POST http_proto=HTTP/1.1 op=chainedandfronted origin=update.getlantern.org origin_host=update.getlanter
                            n.org origin_port=80 root_op=chainedandfronted]
                         E  ERROR go-update.check: check.go:170   at github.com/getlantern/flashlight/vendor/github.com/getlantern/go-update/check.(*Params).CheckForUpdate (check.go:170)
                         E  ERROR go-update.check: check.go:170   at github.com/getlantern/flashlight/vendor/github.com/getlantern/autoupdate.checkUpdate (autoupdate.go:123)
                         E  ERROR go-update.check: check.go:170   at github.com/getlantern/flashlight/vendor/github.com/getlantern/autoupdate.doCheckUpdate (mobile.go:54)
                         E  ERROR go-update.check: check.go:170   at github.com/getlantern/flashlight/vendor/github.com/getlantern/autoupdate.CheckMobileUpdate (mobile.go:82)
                         E  ERROR go-update.check: check.go:170   at github.com/getlantern/flashlight/android.CheckForUpdates (android.go:322)
```